acs-category-theory-notes.tex

```diff
@@ -2497,7 +2506,267 @@ \subsection{Morphisms of monoids, preorders, and categories}
 \end{tabular}
 \end{tabular}\end{center}
 
-\chapter{Natural transformations}
-\lecturedetails{26 October 2017}{M Fiore, \ldots}
+\chapter{Functors continued}
+\lecturedetails{26 October 2017}{M Fiore, N Alcock, Z Yuan}
+
+\section{More about functors}
+
+\begin{definition}
+A functor \(F: \mathcal{A} \rightarrow \mathcal{B}\) specifies a function \(F: \Ob(\mathcal{A}) \rightarrow \Ob(\mathcal{B})\) such that \(\forall X, Y \in \Ob(\mathcal{A})\), there exists \(F_{X, Y}: \mathcal{A}(X, Y) \rightarrow \mathcal{B}(FX, FY)\) such that identities are preserved.
```

Comment:
I think " mapping " instead of " function "? Open to discussion.

Reply:
I think "map" would me more consistent with lecture 6 (see, e.g., page 38 [at the time of my writing this]).